FriendsWave/social/models.py

```diff
@@ -1,3 +1,90 @@
+from functools import _Descriptor
 from django.db import models
+from django.db.models.base import Model
+from django.db.models.fields.related import OneToOneField
+
+class Profil(models.Models):
+    pseudo = models.CharField(max_length=100)
+    image = models.ImageField()
```

Comment:
1- Make sure static file settings if well configure and Specify specific storage of each static data with functionnality